src/ReactBubbleChartD3.js

```diff
@@ -276,7 +277,13 @@ export default class ReactBubbleChartD3 {
           else size = 'large';
           return 'bubble-label ' + size
         })
-        .text(d => d.displayText || d._id)
+        .each(function(d){
+          if (typeof d.displayText === 'object') {
```

Comment:
let's instead add a different property (instead of mooching off of `displayText`). maybe something like `innerComponent` or just `component`. that'd each this if statement to something like:

```js
if (!!d.innerComponent) {
  ReactDOM.render(d.innerComponent);
} else {
  ...
}